app/views/collections/edit.html.erb

```diff
@@ -16,8 +16,14 @@
   </div>
 <% end %>
 
-<h2>Manage Items in this Collection</h2>
-<%= render 'search_collection_dashboard_form'%>
+<br/>
+<br/>
+<div class="row">
+  <h2 class="col-xs-12 col-sm-6"><%= t('Manage Items in this Collection') %></h2>
```

Comment:
I'd guess it also gives a `span.missing_translation`, and might be difficult to make work in the i18n file with all the spaces (and the lack of expected namespacing)?